### PR TITLE
Moredeprecations

### PR DESCRIPTION
**Pull request by andydotxyz**
Remove the remaining deprecations (except widget.AppTabs).
Some minor coding involved because Box was still in use and a little magic...

Progresses #1724

### Checklist:

- [x] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.
- [x] Any breaking changes have a deprecation path or have been discussed.

### CODE REVIEW

File: CHANGELOG.md
Comment (Jacalz) [inline CHANGELOG.md:39]: It’s widget.Card, isn’t it?

**Review by Jacalz — CHANGES_REQUESTED** [review]
One little typo mostly. Should we perhaps add the button.HideShadow and button.Style removals to the Changelog?

File: CHANGELOG.md
Comment (andydotxyz) [reply CHANGELOG.md]: oops :)

Fixed the other comment with a new line too - thanks

**Comment by Jacalz** [comment]
There also looks to be compile errors.

**Comment by andydotxyz** [comment]
> There also looks to be compile errors.

It's refering to code not on the branch, I don't know what is going on

**Review by Jacalz — CHANGES_REQUESTED** [review]
The selection of primary colour seems to be gone from `fyne_settings`.

**Comment by andydotxyz** [comment]
> The selection of primary colour seems to be gone from `fyne_settings`.

Yeah I noticed that too sorry, bad Box removal. Fixed again.

**Comment by andydotxyz** [comment]
Phew finally fixed it @Jacalz :)

**Review by Jacalz — CHANGES_REQUESTED** [review]
Sorry for missing this earlier, but you probably would want to get `cmd/fyne_settings/screenshot.png` updated while you are at it.

**Review by Jacalz — COMMENTED** [review]
It is looking great otherwise. Ping me on slack and I will get this approved once that lands :+1:

**Review by Jacalz — APPROVED** [review]
There are conflicts, but looks good to me otherwise 👍

**Review by Jacalz — APPROVED** [review]

**Pull request merged**
